### PR TITLE
Audification multiple legend items support

### PR DESCRIPTION
**Pull request by 64json**

### CODE REVIEW

File: src/components/card/card.component.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
   display: flex;
   flex-direction: column;
   align-items: stretch;
+  width: 854px;
```
Comment (Awk34) [inline src/components/card/card.component.scss:7]: magic number?

File: src/components/line-chart-audification/line-chart-audification.component.spec.ts [hunk 2]
```diff
@@ -106,6 +102,14 @@ describe('LineChartAudificationComponent', () => {
     }
   });
 
+  it('should switch the legend item upon pressing UP or DOWN.', () => {
+    ['ArrowUp', 'ArrowDown'].forEach(key => {
+      const prevDatumIndex = component.datumIndex;
+      triggerKeyDown(key);
+      expect(component.datumIndex).not.toBe(prevDatumIndex);
```
Comment (Awk34) [inline src/components/line-chart-audification/line-chart-audification.component.spec.ts:109]: should you also be checking that the legend item changes focus?

File: src/i18n/utils.ts [hunk 3]
```diff
@@ -32,5 +32,10 @@ export function t(key: I18nKey, ...args: any[]) {
  * @param args The arguments to format the string with.
  */
 export function tA11y(key: I18nKey, ...args: any[]) {
-  return striptags(t(key, ...args));
+  return striptags(
+    t(key, ...args)
+      .replace(/\.{3}/g, 'to')
+      .replace(/<kbd>(\S+)<\/kbd>/g, `'$1'`)
+      .replace(/'\?'/g, `'QUESTION MARK'`),
```
Comment (Awk34) [inline src/i18n/utils.ts:39]: Is it a problem that these are always english?

File: src/components/card/card.component.scss (hunk 1 above)
Comment (64json) [reply src/components/card/card.component.scss:7]: I'll make a variable for the magic number.

File: src/i18n/utils.ts (hunk 3 above)
Comment (64json) [reply src/i18n/utils.ts:39]: oh I think it should be in i18n indeed.

File: src/components/line-chart-audification/line-chart-audification.component.spec.ts (hunk 2 above)
Comment (64json) [reply src/components/line-chart-audification/line-chart-audification.component.spec.ts:109]: Which element do you mean by the legend item?

Comment (Awk34) [reply src/components/line-chart-audification/line-chart-audification.component.spec.ts:109]: The test describes checking that UP/DOWN changes the legend item, but it looks like the test just verifies that new index is not the old index. Should it maybe verify that the new index is the correct to focus?

**Pull request merged**